pennylane/devices/default_gaussian.py

```diff
@@ -656,8 +656,6 @@ class DefaultGaussian(Device):
     version = "0.12.0"
     author = "Xanadu Inc."
 
-    _capabilities = {"model": "cv"}
```

Comment:
Just double checking, but as far as I can tell, devices can (if they want to) simply keep defining the `_capabilities` attribute, and things will continue to work as before? That is, this change in `default.gaussian` isn't _required_ to avoid breakages?

Reply:
Good point...in theory yes, the external devices simply inherit/overwrite the base classes' ``_capabilities``.

However, I ran the device tests on some of them, and there are a few issues:
* The rename from `inverse_operations` to `supports_inverse_operations` means that even though a device says it supports inverse operations, it registers it under the wrong keyword and the core will not know it. We could fix this by revoking the change for now.
* `QubitDevice` is set by default to support sampled and exact mode. This needs to be adapted for every device inheriting from it. However, these capabilities are only used in the device tests, so core should still work.

Reply:
Hmm, in the interest of not breaking anything I will change all core calls to these three keys so that they also look for the old versions.

Reply:
Still a bit confused about this change here. Should it rewrite `_capabilities` that is being inherited with `"model": "cv"` in it?